buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1625,6 +1645,110 @@ class BeamModulePlugin implements Plugin<Project> {
 
     /** ***********************************************************************************************/
 
+    // Method to create the crossLanguageValidatesRunnerTask.
+    // The method takes crossLanguageValidatesRunnerConfiguration as parameter.
+    project.ext.createCrossLanguageValidatesRunnerTask = {
```

Comment:
Can we add new test suites that run this task for various runners (to https://github.com/apache/beam/tree/master/.test-infra/jenkins). We can start with Flink and add direct runner as well if that works.

Reply:
added the postcommit test.